src/utils/api.js

```diff
@@ -4,11 +4,17 @@ const newsApi = axios.create({
     baseURL: "https://nc-news-dw.onrender.com/api"
 })
 
-export const fetchArticles = (topic_name) => {
+export const fetchArticles = (topic_name, sort_by, order) => {
     const params = {}
     if(topic_name){
         params.topic = topic_name
     }
+    if(sort_by){
```

Comment:
You can just put those properties into the params object, as when they are undefined, they just get ignored anyways..